unittests/testdictmerge.py

```diff
@@ -80,3 +80,329 @@ def test_dotdict(self):
             {"key": "value", "sub.subkey": "subvalue"},
             msg="Dotdict returns incorrect result",
         )
+
+    def test_assets_ext_attr(self):
+        test_data_assets = [
+            {
+                "identity": "assets/525e4c24-711c-457d-b528-f9a4879f0759",
+                "behaviours": [
+                    "Builtin",
+                    "AssetCreator",
+                    "RecordEvidence",
+                    "Attachments",
+                ],
+                "attributes": {
+                    "arc_display_name": "Asset with attachment",
+                    "arc_display_type": "Nested attribute",
+                    "arc_attachments": [
+                        {
+                            "arc_hash_value": "b6ec2d1829b8fbf0d58c4f4d87a56a68b16236ef173c7b8b23713078010049ff",
+                            "mime_type": "application/pdf",
+                            "arc_attachment_identity": "blobs/ea4bcf42-2b72-4903-919b-b856443fc511",
+                            "arc_display_name": "Inspection Standards",
+                            "arc_hash_alg": "SHA256",
+                        }
+                    ],
+                    "arc_description": "Asset created for testing",
+                },
+                "confirmation_status": "CONFIRMED",
+                "tracked": "TRACKED",
+                "owner": "##########",
+                "at_time": "2022-06-13T22:57:54Z",
+                "storage_integrity": "TENANT_STORAGE",
+                "proof_mechanism": "SIMPLE_HASH",
+                "chain_id": "##########",
+            },
+            {
+                "identity": "assets/976f03e8-458d-42fb-9cc4-720720ed648f",
+                "behaviours": [
+                    "Builtin",
+                    "AssetCreator",
+                    "RecordEvidence",
+                    "Attachments",
+                ],
+                "attributes": {
+                    "arc_display_type": "Test",
+                    "Weight": "1192kg",
+                    "arc_description": "Asset created for testing",
+                    "arc_display_name": "Test Asset",
+                },
+                "confirmation_status": "CONFIRMED",
+                "tracked": "TRACKED",
+                "owner": "##########",
+                "at_time": "2022-06-13T22:57:54Z",
+                "storage_integrity": "TENANT_STORAGE",
+                "proof_mechanism": "SIMPLE_HASH",
+                "chain_id": "##########",
+            },
+        ]
+        self.assertEqual(
+            len(dictmerge.assets_ext_attr(test_data_assets)),
+            1,
+            msg="assets_ext_attr returns incorrect result",
+        )
```

Comment:
test expected out put as well

         result = dictmerge.assets_ext_attr(test_data_assets)
         self.assertEqual(
            len(result),
            1,
            msg="assets_ext_attr returns incorrect result",
        )
        self.assertEqual(
            result,
            EXPECTED_ASSETS_EXT_ATTR,   # defined at top of file
            msg="assets_ext_attr returns incorrect result",
        )